public/stylesheets/site/2.0/13-group-blurb.css

```diff
@@ -368,12 +368,16 @@ blurbs on the manage collection items pages, mostly reseting styles inherited fr
 }
 
 .bookmark .short .status {
-	left: 30px;
+	left: 0;
 	margin-top: 0;
 	top: 0;
 	width: 25px;
 }
 
+.bookmark .short .header h5 {
```

Comment:
(This probably would've worked without .header, but we don't want to accidentally affect users' h5s in bookmark notes.)